cmd/opm/alpha/veneer/render.go

```diff
@@ -0,0 +1,83 @@
+package veneer
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/veneer/basic"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
+	containerd "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+)
+
+func newBasicVeneerRenderCmd() *cobra.Command {
+	var (
+		veneer basic.Veneer
+		output string
+	)
+	cmd := &cobra.Command{
+		Use:   "render veneer-file",
+		Short: "Generate a declarative config blob from a single 'basic veneer' file",
+		Long:  `Generate a declarative config blob from a single 'basic veneer' file, typified as a declarative configuration file where olm.bundle objects have no properties`,
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+
+			var write func(declcfg.DeclarativeConfig, io.Writer) error
+			switch output {
+			case "yaml":
+				write = declcfg.WriteYAML
+			case "json":
+				write = declcfg.WriteJSON
+			default:
+				log.Fatalf("invalid --output value %q, expected (json|yaml)", output)
+			}
+
+			// The bundle loading impl is somewhat verbose, even on the happy path,
+			// so discard all logrus default logger logs. Any important failures will be
+			// returned from veneer.Ingest and logged as fatal errors.
```

Comment:
```suggestion
			// returned from veneer.Render and logged as fatal errors.
```

Reply:
Dangit.  I knew I had missed one, but couldn't find it. Thanks.